cdc/sink/manager.go

```diff
@@ -85,7 +86,18 @@ func (m *Manager) Close(ctx context.Context) error {
 	defer m.tableSinksMu.Unlock()
 	tableSinkTotalRowsCountCounter.DeleteLabelValues(m.captureAddr, m.changefeedID)
 	if m.bufSink != nil {
-		return m.bufSink.Close(ctx)
+		log.Info("sinkManager try close bufSink",
```

Comment:
May be we should add logs in processor `manager.closeProcessor` and indicate which changefeed we are closing.

Reply:
we can add a method named `timeit` to record all time consuming operations. maybe it would look like 

```go
func timeit(func() error) error
```